packages/mds-policy-service/@types/intents.ts

```diff
@@ -0,0 +1,74 @@
+import { now } from '@mds-core/mds-utils'
+import type { BaseRule, MicroMobilityStatesToEvents, PolicyDomainModel } from './models'
+
+export const INTENT_TYPES = ['no_parking'] as const
```

Comment:
I tend to prefer `<const>` over `as const` these days....
```suggestion
export const INTENT_TYPES = <const>['no_parking']
```